pyrefly/lib/config/config.rs

```diff
@@ -802,7 +816,7 @@ mod tests {
                         (ErrorKind::AssertType, true),
                         (ErrorKind::BadReturn, false)
                     ]))),
-                    ignore_errors_in_generated_code: Some(true),
+                    ignore_errors_in_generated_code: None,
```

Comment:
I'm not sure about this change as I don't understand why the field  `ignore_errors_in_generated_code` of the root config was `Some(true)` in the first place: [by looking at ConfigBase definition](https://github.com/facebook/pyrefly/blob/27f4e8a0f2103094ede6ba8ba3b558bb7a1892e1/pyrefly/lib/config/base.rs#L61) I would have expected it to be `None` by default.

I would appreciate if someone could enlighten me on this matter :)

Reply:
My bad, I did not understand that I had to set the ` ignore_errors_in_generated_code` field of the root config instead of create a new `ignore_errors_in_generated_code` field on ConfigFile.